index.js

```diff
@@ -10,7 +10,8 @@ var noop = function() {}
 module.exports = function(db) {
   var feed = {}
   var lock = mutexify()
- 
+  var valueEncoding = db.options.valueEncoding || 'binary'
```

Comment:
right, so maybe `db.options.valueEncoding || options.valueEncoding || 'binary'` would be better